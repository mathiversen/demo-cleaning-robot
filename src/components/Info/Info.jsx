import React from 'react'
import { css } from 'goober'

export default ({ children }) => {
    const className = css`
        display: grid;
        grid-gap: 1rem;
        grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
        place-items: center;
        background: black;
        color: white;
        padding: 1rem;
    `

    return <p className={className}>{children}</p>
}
