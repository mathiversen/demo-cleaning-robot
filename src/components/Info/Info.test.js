import React from 'react'
import Info from './Info'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
    const mockProps = {
        children: <div key="x">child</div>
    }
    const tree = renderer.create(<Info {...mockProps} />).toJSON()
    expect(tree).toMatchSnapshot()
})
