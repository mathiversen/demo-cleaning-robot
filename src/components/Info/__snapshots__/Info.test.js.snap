// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="go405397641"
>
  <div>
    child
  </div>
</p>
`;
